molr-commons/src/main/java/io/molr/commons/domain/Out.java

```diff
@@ -8,4 +10,5 @@
 
     <T> void emit(Placeholder<T> placeholder, T value);
 
+    <T> void emit(Placeholder<T> placeholder, Collection<T> value);
```

Comment:
This for sure is wrong! has to be Placeholder<Collection<T>>

Reply:
Indeed with TypeToken you do not want to start here ;-)
I fully agree with michi, the criterion should always be how it transports over json. E.g. there is already the problem with Long and Integer ... we only have one of them (I do not remember), since you cannot know on the other hand where you should cast to ...)

Cf. Michi: Indeed we cannot allow arbitrary java types. We have to agree e.g. on one of:
* List<String>
* String[]
... as both of them are transformed into a json list (probably even others ... e.g. Set ... I am not sure)

... I think for such proposals, we should always do testing with remote moles right before the pull request... And preferrable also extending the python mole to see how it maps to other languages ...